tests/unit/lms/tasks/task_done_test.py

```diff
@@ -0,0 +1,41 @@
+from contextlib import contextmanager
+from datetime import datetime, timedelta
+
+import pytest
+from freezegun import freeze_time
+from sqlalchemy import select
+
+from lms.models import TaskDone
+from lms.tasks.task_done import delete_expired_rows
+
+
+@freeze_time("2023-05-04 12:12:01")
+def test_delete_expired_rows(db_session):
+    frozen_time = datetime.fromisoformat("2023-05-04 12:12:01")
```

Comment:
You're supposed to be able to get `freezegun` to pass the `frozen_time` argument to your test like this:

```python
@freeze_time("2012-01-14", as_arg=True)
def test(frozen_time):
    ...
```

But that doesn't seem to work with `pytest`: `fixture 'frozen_time' not found`.